radicale/__init__.py

```diff
@@ -92,6 +93,14 @@ def __init__(self, address, handler):
         self.server_activate()
 
 
+class ThreadedHTTPServer(socketserver.ThreadingMixIn, HTTPServer):
```

Comment:
Unsure if daemon_thread = True should be set. If I blast the server with requests, it can't be quit properly.